crates/proto/proto/responses.proto

```diff
@@ -32,6 +32,12 @@ message NullifierUpdate {
     fixed32 block_num = 2;
 }
 
+message TransactionUpdate {
+    digest.Digest transaction_id = 1;
+    fixed32 block_num = 2;
+    fixed64 account_id = 3;
+}
```

Comment:
I wonder if `TransactionSummary` would be a better name here (would be parallel with `AccountSummary`).

Also, as discussed in the issue, let's create a separate message for `TransactionId`.

Reply:
done! One kind of annoying thing I learned is that proto3 doesn't have a required field, which makes it kind of annoying to have all the `Option`s laying around 😅